addons/website/static/src/client_actions/website_preview/website_preview_test_mode.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+import { patch } from 'web.utils';
+import { WebsitePreview } from '@website/client_actions/website_preview/website_preview';
+
+patch(WebsitePreview.prototype, 'website_preview_test_mode', {
+    /**
+     * @override
+     */
+    get testMode() {
+        return true;
+    }
```

Comment:
Would not it be enough to check if debug == tests ?

Reply:
Good to know :+1:

Reply:
(although should not we have that value in session anyway instead of having to add conditional files to do it ? :grimacing: